SmartCarMQTTApp/app/src/main/java/com/example/smartcarmqttapp/screens/quiz/PracticeTheoryActivity.java

```diff
@@ -224,7 +228,7 @@ private void addModesToModeListView() {
                 numOfQuestionsTextView.setVisibility(View.VISIBLE);
                 findViewById(R.id.cardCategory).setVisibility(View.VISIBLE);
                 MILLIS = 0;
-                numOfQuestions = 15;
+                numOfQuestions = 45;
```

Comment:
maybe create a constant for 45 at the top of the file, so it is easier to find, understand, and change, than when it gets assigned on a line 231 ;)

Reply:
Will do good catch!!!